doc/api/tls.md

```diff
@@ -911,10 +913,18 @@ added: v0.11.13
     the same order as their private keys in `key`. If the intermediate
     certificates are not provided, the peer will not be able to validate the
     certificate, and the handshake will fail.
-  * `ca`{string|string[]|Buffer|Buffer[]} Optional CA certificates to trust.
-    Default is the well-known CAs from Mozilla. When connecting to peers that
-    use certificates issued privately, or self-signed, the private root CA or
-    self-signed certificate must be provided to verify the peer.
+  * `ca` {string|string[]|Buffer|Buffer[]} Optionally specify the trusted CA
+    certificates. Default is to trust the well-known CAs curated by Mozilla.
+    Mozilla's CAs are completely replaced when CAs are explicitly specified
+    using this option.
+    Multiple CAs can be provided using an array, and any string or Buffer can
+    contain multiple PEM CAs concatenated together. The peer's certificate must
+    be chainable to a CA trusted by the server for the connection to be
```

Comment:
>The peer's certificate must be chainable to a CA trusted by the server

Does `a CA trusted by the server` mean one of the CAs provided in this option? If so maybe:

 `to a CA trusted by the server` -> `to one of the provided CAs`